docs/src/content/docs/guides/make-onetime-payment.mdx

```diff
@@ -121,28 +98,140 @@ Add the following properties:
+:::note[Limits]
+
+The optional `debitAmount` and `receiveAmount` parameters, which apply to the sender and recipient respectively, allow you to define the limits under which an outgoing payment can be created.
 
-For details on how to facilitate an interactive grant refer to the <LinkOut href='https://openpayments.dev/guides/create-interactive-grant/'>
-Create an Interactive Grant guide. </LinkOut>
+The value for either parameter is the amount that all payments created under the grant cannot exceed. For example, the `DEBIT_AMOUNT` value is the maximum amount that can be debited from the payer's wallet.
 :::
 
+Next, indicate your client is capable of directing the end-user to a URI to start an interaction.
+
+1. Add the `interact` object to your grant request.
+2. Specify `redirect` as the `start` mode. Open Payments only supports `redirect` at this time. The redirect URI will be provided by the authorization server in its response.
+
+Now, indicate your client can receive a signal from the authorization server when the interaction has finished.
+
+1. Add the `finish` object within the `interact` object.
+2. Specify `redirect` as the `method`. Open Payments only supports `redirect` at this time.
+3. Specify the `uri` that the authorization server will send the end-user back to when the interaction has finished.
+4. Create a `nonce` for your client to use to verify the authorization server is the same entity throughout the entire process. The nonce can be any random, hard-to-guess string.
+
 <ChunkedSnippet
   source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
   chunk='4'
 />
 
-### 8. Create an Outgoing Payment
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='5'
+/>
+
+### 6. Start interaction with the end user
+
+Once your client receives the authorization server’s response, it must send the end-user to the `interact.redirect` URI contained in the response. This starts the interaction flow.
+
+The response also includes a `continue` object, which is essential for managing the interaction and obtaining explicit user consent for outgoing payment grants. The `continue` object contains an access token and a URI that the client will use to finalize the grant request after the user has completed their interaction with the identity provider (IdP). This ensures that the client can securely obtain the necessary permissions to proceed with the payment process.
+
+<CodeBlock title="Example response">
+```json
+{
+  "interact": {
+    "redirect": "https://auth.interledger-test.dev/4CF492MLVMSW9MKMXKHQ",
+    "finish": "4105340a-05eb-4290-8739-f9e2b463bfa7"
+  },
+  "continue": {
+    "access_token": {
+      "value": "33OMUKMKSKU80UPRY5NM"
+    },
+    "uri": "https://auth.interledger-test.dev/continue/4CF492MLVMSW9MKMXKHQ",
+    "wait": 30
+  }
+}
+```
+</CodeBlock>
+
+:::note
```

Comment:
I don't think we need this note here. Its contents are already mentioned in step 5 and step 7.